package.json

```diff
@@ -15,7 +15,9 @@
     "pretest": "./scripts/pretest.sh",
     "test-nocoverage": "NODE_ENV=test ./node_modules/.bin/mocha --delay --compilers js:babel-register test/test_*.js",
     "test-watch": "nodemon -q -x npm test",
-    "test": "./node_modules/.bin/nyc npm run test-nocoverage && npm run lint"
+    "test": "./node_modules/.bin/nyc npm run test-nocoverage && npm run lint",
+    "gh-pages-travis": "gh-pages-travis",
+    "check-failures": "./scripts/OGC_tests/check_tests_failures.sh"
```

Comment:
No need for this. We can simply call `./scripts/OGC_tests/check_tests_failures.sh` from the `.travis.yml` file like we do with the other scripts.